plane/fit_multi_plane.py

```diff
@@ -1,43 +1,81 @@
 import numpy as np
-from parse_xml import ParseXML
+# from plane.fit_plane import FitPlane
+from plane.parse_xml import ParseXML
+import pandas as pd
 
 class FitMultiPlane:
-    def __init__(self):
+    def __init__(self, fitplanes, real_centers, template_size, um_per_pixel):
         """
         - list of barcodes contained in object
             - contains self.scale, self.theta_deg, self.shear_magnitude, self.shear_vector, self.tx, self.ty, self.z
             - Real positions of each barcode center as defined by photobleach script
         - Adjacency matrix to store distances between pairs of barcodes.
         - ??? to store angles between 3 barcodes for xy, xz, yz planes 
         """
-        pass
+        self.fitplanes = fitplanes
+        self.real_centers = real_centers # in um
+        self.template_size = template_size
+        self.um_per_pixel = um_per_pixel
+        self.fitplane_centers = self.calc_fitplane_centers() # in um
+        self.distances = self.calc_distances() # in um
 
     @classmethod
-    def from_imagej_xml(cls):
+    def from_aligned_fitplanes(cls, fitplanes_list, real_centers_list, template_size=401, um_per_pixel=2):
```

Comment:
This interface is a bit messy. 
I suggest we swap to a more clean interface:
template_centers_physical_position_xyz_um - a numpy array (n by 3) of the center position for each template in microns. 
This variable will combine real_center_list for the xy position and z position we will need to get from user.

template_centers_image_position_uv_pix - the position of each center in the image in pixels.
We get that number from fit_plane

Given this interface, I think we can skip template_size and um_per_pixel and not pass them as inputs right?

Reply:
I understand that this change may break some of the functions below. But we will need to figure this out later